internal/infra/redis_infra.go

```diff
@@ -0,0 +1,103 @@
+package infra
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v8"
+)
+
+type RedisConnector struct {
+	Clinet *redis.Client
+}
+
+func (r *RedisConnector) CreateUser(ctx context.Context, username string, password string, db int) error {
+	isExist, checkUserErr := r.IsUserExist(ctx, username)
+	if checkUserErr != nil {
+		return checkUserErr
+	}
+
+	if isExist {
+		return fmt.Errorf(fmt.Sprintf("duplicate user: %s", username))
+	}
+
+	cmds := []interface{}{
+		"acl",
+		"setuser",
+		username,
+		fmt.Sprintf(">%s", password),
+		"on",
+		"+@all",
+		"allchannels",
+		"allkeys",
+		"-acl",
+		"-bgrewriteaof",
+		"-bgsave",
+		"-config",
+		"-module",
+	}
+
+	if db >= 0 {
+		cmds = append(cmds, "-select", fmt.Sprintf("+select|%v", db))
```

Comment:
%d for int

Reply:
[35aba8c](https://github.com/pnetwork/rapd-tf-provider-redis/pull/1/commits/35aba8c5ec7f426de502c3f7c582fa736e79e5f8)